templates/aws-tap-entrypoint-existing-vpc.template.yaml

```diff
@@ -703,211 +695,6 @@ Conditions:
-          OidcProviderEndpoint: !Select [1, !Split [//, !GetAtt ITERATEEKSQSStack.Outputs.OIDCIssuerURL]]
-      ManagedPolicyArns:
-        - arn:aws:iam::aws:policy/service-role/AmazonEBSCSIDriverPolicy
-  SINGLEEKSEBSCSIDriverRole:
-    Type: AWS::IAM::Role
-    Condition: CreateSingleCluster
-    Properties:
-      RoleName: !Sub
-        - SINGLEEKSEBSCSIDriverRole-${StackId}
-        - StackId: !Select [2, !Split [/, !Ref AWS::StackId]]
-      Description: >-
-        VMware Tanzu Application Platform role for EBSSCI Driver.
-      AssumeRolePolicyDocument: !Sub
-        - |
-          {
-            "Version": "2012-10-17",
-            "Statement": [
-              {
-                "Effect": "Allow",
-                "Principal": {
-                  "Federated": "${IamOidcProviderArn}/${OidcProviderEndpoint}"
-                },
-                "Action": "sts:AssumeRoleWithWebIdentity",
-                "Condition": {
-                  "StringEquals": {
-                    "${OidcProviderEndpoint}:aud": "sts.amazonaws.com",
-                    "${OidcProviderEndpoint}:sub": "system:serviceaccount:kube-system:ebs-csi-controller-sa"
-                  }
-                }
-              }
-            ]
-          }
-        - IamOidcProviderArn: !Sub arn:aws:iam::${AWS::AccountId}:oidc-provider
-          OidcProviderEndpoint: !Select [1, !Split [//, !GetAtt EKSQSStack.Outputs.OIDCIssuerURL]]
-      ManagedPolicyArns:
-        - arn:aws:iam::aws:policy/service-role/AmazonEBSCSIDriverPolicy
-  BUILDEBSSCIAddon:
-    Type: AWS::EKS::Addon
-    Condition: CreateMultiCluster
-    Properties:
-      AddonName: aws-ebs-csi-driver
-      ClusterName: !Join ['', [!Ref EKSClusterName, "-build" ]]
-      ResolveConflicts: NONE
-      ServiceAccountRoleArn: !GetAtt BUILDEKSEBSCSIDriverRole.Arn
-  RUNEBSSCIAddon:
-    Type: AWS::EKS::Addon
-    Condition: CreateMultiCluster
-    Properties:
-      AddonName: aws-ebs-csi-driver
-      ClusterName: !Join ['', [!Ref EKSClusterName, "-run" ]]
-      ResolveConflicts: NONE
-      ServiceAccountRoleArn: !GetAtt RUNEKSEBSCSIDriverRole.Arn
-  VIEWEBSSCIAddon:
-    Type: AWS::EKS::Addon
-    Condition: CreateMultiCluster
-    Properties:
-      AddonName: aws-ebs-csi-driver
-      ClusterName: !Join ['', [!Ref EKSClusterName, "-view" ]]
-      ResolveConflicts: NONE
-      ServiceAccountRoleArn: !GetAtt VIEWEKSEBSCSIDriverRole.Arn
-  ITERATEEBSSCIAddon:
-    Type: AWS::EKS::Addon
-    Condition: CreateMultiCluster
-    Properties:
-      AddonName: aws-ebs-csi-driver
-      ClusterName: !Join ['', [!Ref EKSClusterName, "-iterate" ]]
-      ResolveConflicts: NONE
-      ServiceAccountRoleArn: !GetAtt ITERATEEKSEBSCSIDriverRole.Arn
-  SINGLEEBSSCIAddon:
-    Type: AWS::EKS::Addon
-    Condition: CreateSingleCluster
-    Properties:
-      AddonName: aws-ebs-csi-driver
-      ClusterName: !Ref EKSClusterName
-      ResolveConflicts: NONE
-      ServiceAccountRoleArn: !GetAtt SINGLEEKSEBSCSIDriverRole.Arn
```

Comment:
I'm a bit confused with this block of changes. Does 1.24 install the CSI driver by default?

Reply:
This is now part of https://github.com/aws-quickstart/quickstart-amazon-eks.
It came in with https://github.com/aws-quickstart/quickstart-amazon-eks/commit/f3b5980c84c93b198184eb653b6548d4677ff69d and got a shout out in https://github.com/aws-quickstart/quickstart-amazon-eks/commit/5a43fc3c8073d7b6f706c425dcd002e4363270a8.

So the EKS stack brings this now, and with bumping the version of `submodules/quickstart-amazon-eks` we don't have to do that ourselves anymore.

Reply:
Those lines also need to be removed from other clusters.

```
  BUILDEBSSCIAddon:
    Type: AWS::EKS::Addon
    Condition: CreateMultiCluster
    Properties:
      AddonName: aws-ebs-csi-driver
      ClusterName: !Join ['', [!Ref EKSClusterName, "-build" ]]
      ResolveConflicts: NONE
      ServiceAccountRoleArn: !GetAtt BUILDEKSEBSCSIDriverRole.Arn
  RUNEBSSCIAddon:
    Type: AWS::EKS::Addon
    Condition: CreateMultiCluster
    Properties:
      AddonName: aws-ebs-csi-driver
      ClusterName: !Join ['', [!Ref EKSClusterName, "-run" ]]
      ResolveConflicts: NONE
      ServiceAccountRoleArn: !GetAtt RUNEKSEBSCSIDriverRole.Arn
  VIEWEBSSCIAddon:
    Type: AWS::EKS::Addon
    Condition: CreateMultiCluster
    Properties:
      AddonName: aws-ebs-csi-driver
      ClusterName: !Join ['', [!Ref EKSClusterName, "-view" ]]
      ResolveConflicts: NONE
      ServiceAccountRoleArn: !GetAtt VIEWEKSEBSCSIDriverRole.Arn
  ITERATEEBSSCIAddon:
    Type: AWS::EKS::Addon
    Condition: CreateMultiCluster
    Properties:
      AddonName: aws-ebs-csi-driver
      ClusterName: !Join ['', [!Ref EKSClusterName, "-iterate" ]]
      ResolveConflicts: NONE
      ServiceAccountRoleArn: !GetAtt ITERATEEKSEBSCSIDriverRole.Arn
```